microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -44,7 +44,9 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. In the **Publish APIs** page, select the **Create API credentials** button to generate the API credentials.  This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
 
-1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**.  You'll use them in the next step to get an access token.
+1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**. You'll use them in the next step to get an access token. Note that client secret is visible only once, after enabling or renewing the API. 
+
+![image](https://user-images.githubusercontent.com/11265822/150777593-42cb9d37-0c1f-4ef4-9de6-9c4bc1166b96.png)
```

Comment:
* Should this image be copied into this repo?  
* The alt-text should be descriptive enough to differentiate the image from other images, and should end with a period.
* There should be introductory text or some transition/glue from the lead-in text to the image.  A test is: when reading this .md file without looking at the image & rendered article, a writer should be able to glean what the image shows.